layers/meta-tegra-support/recipes-ota/packagegroups/packagegroup-demo-base.bbappend

```diff
@@ -0,0 +1,8 @@
+RDEPENDS_${PN}_append = " \
+    gptfdisk \
+    i2c-tools \
+    tegra-bup-payload \
+    tegra-eeprom-tool \
+    tegra-fuse-tool \   
+    tegra-sysinstall-tools \
+"
```

Comment:
I'm not sure we really want to include all of these packages in every image.  Can we drop this bbappend?